api/jobs.py

```diff
@@ -2,14 +2,20 @@
 API request handlers for process-job-handling.
 """
 
+# We shadow the standard library; this is a workaround.
+from __future__ import absolute_import
```

Comment:
We have a python package that is unambiguously named relative to the standard library - stdlib has `collections`, we have `collections.py`. Apparently in python 3 this is less painful.

That snippet is a backport of smarter import behaviour, changing the file to actually do what we ask it to `from collections import namedtuple` rather than interpreting that as `from ./collections.py import namedtuple`.